scripts/setModuleDetails.js

```diff
@@ -0,0 +1,45 @@
+import ora from "ora";
+import { invalid, section, valid } from "../utils.js";
+import { apiClient } from "./utils/apiClient.js";
+
+export const setModuleDetails = async (
+  id, name, description, searchDescription, acceptanceCriteria
+) => {
+  const patchBody = {};
+
+  section("To ensure the entire string is saved, please use double quotes around the string value.");
```

Comment:
I think we could use some validation here to ensure that at least one value is being set. Like

`if(Object.keys(patchBody).length === 0)`

Then we can print this warning inside there if they didnt provide any values to set.